las/client.py

```diff
@@ -588,27 +684,51 @@ def get_document(self, document_id: str) -> Dict:
         """
         return self._make_request(requests.get, f'/documents/{document_id}')
 
-    def update_document(self, document_id: str, ground_truth: Sequence[Dict[str, Union[Optional[str], bool]]]) -> Dict:
+    def update_document(
+        self,
+        document_id: str,
+        *,
```

Comment:
This is potentially a breaking change. Users who have not used kwarg for ground_truth will get an error moving forward. Do we want this?

Reply:
no, we do not for now.